e2e_tests/test_workspace_templates.py

```diff
@@ -36,3 +38,20 @@ async def test_get_workspace_template(template_name, verify) -> None:
     admin_token = await get_admin_token(verify)
     async with get_template(template_name, strings.API_WORKSPACE_TEMPLATES, admin_token, verify) as response:
         assert_status(response, [status.HTTP_200_OK], f"Failed to GET template: {template_name}")
+
+
+@pytest.mark.extended
+@pytest.mark.parametrize("template_name", workspace_templates)
```

Comment:
workspace_templates include the base one too which is anyway checked as part of the guacamole tests. I think you should somehow split it how to save 30 mins of tests.

Reply:
Is that any better?